content/consul/v1.21.x/content/docs/secure/sso/auth0.mdx

```diff
@@ -143,11 +74,7 @@ Edit the user's record and assign some `user_metadata` and `app_metadata` such a
 
 ![Auth0 UI configure user metadata](/img/sso/auth0-configure-user-metadata.png)
```

Comment:
````suggestion
The data you associate with Auth0 may change depending on your existing configurations. This example uses `user_metadata` and `app_metadata` to configure Auth0.

```json
{
// user_metadata (end-user editable)
"first_name": "Kara",
"last_name": "Danvers"
}
```

```json
{
// app_metadata (end-user not editable)
"roles": ["engineering"]
}
```
````